Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Docker 17.05 or higher required for multi-stage builds
+FROM rust:1.26.0-stretch as builder
+ADD . /app
+WORKDIR /app
+# Make sure that this matches in .travis.yml
+ARG RUST_TOOLCHAIN=nightly
+RUN \
+    apt-get -qq update && \
+    apt-get -qq install -y default-libmysqlclient-dev && \
+    \
+    rustup default ${RUST_TOOLCHAIN} && \
+    cargo --version && \
+    rustc --version && \
+    # mkdir -m 755 bin && \  TODO: Do we move config to the binary or binary to the config??????
```

Comment:
probably move the config to it due to the huge size of image otherwise: 

<img width=300 src=https://user-images.githubusercontent.com/128755/41696453-36c7cb36-74e2-11e8-8e5b-eed0633d0d29.png />